packages/core-data/src/entity-provider.js

```diff
@@ -153,17 +155,19 @@ export function useEntityProp( kind, name, prop, _id ) {
 export function useEntityBlockEditor( kind, name, { id: _id } = {} ) {
 	const providerId = useEntityId( kind, name );
 	const id = _id ?? providerId;
-	const { content, editedBlocks, meta } = useSelect(
+	const { content, editedBlocks, meta, entityRecord } = useSelect(
 		( select ) => {
 			if ( ! id ) {
 				return {};
 			}
-			const { getEditedEntityRecord } = select( STORE_NAME );
+			const { getEditedEntityRecord, getEntityRecord } =
+				select( STORE_NAME );
 			const editedRecord = getEditedEntityRecord( kind, name, id );
 			return {
 				editedBlocks: editedRecord.blocks,
 				content: editedRecord.content,
 				meta: editedRecord.meta,
+				entityRecord: getEntityRecord( kind, name, id ),
```

Comment:
I think we don't really need the `entityRecord` to trigger a pre-render, we should just use `getEntityRecord` within the blocks computation.

Reply:
You mean like this? [b332cf9](https://github.com/WordPress/gutenberg/pull/58146/commits/b332cf9dc132db793e4308a508a8bae860a4b8f9)

I think it doesn't matter much because the entityRecord will never change, unlike the edited entity record, which is why we use it as a cache key.